### ISSUE TITLE
Line Highlights Size

### ISSUE THREAD

**Issue by duplaja** [PrismJS/prism#1036]
Is there a reason that the div used for line-highlights does not auto-size to the same as the code block?

I added this in (minified version), to let it do that:   h.style.width=e.firstChild.offsetWidth+60+'px',

Otherwise, on long lines, you can scroll and then highlight stops at the width of the pre tag. You can see this behavior on the examples on PrismJS.com

**Comment by jarodium** [comment]
If is this you are refering to, then I got the same issue.
This was the result after I linked with #id-div.2-5

Also, I'm not sure if this helps, but I am using the code highlighter inside a second level tabbed page on bootstrap, which is not loaded at page loading. Basically I load the code highlighter after a ajax request.
 
![captura de ecra 2017-01-28 as 16 02 45](https://cloud.githubusercontent.com/assets/1470446/22397869/475e14c8-e573-11e6-9afb-f9fafad29877.png)

**Comment by chenningg** [comment]
Still having this issue. Is there a fix for the unminified javascript?

**Comment by chenningg** [comment]
Managed to fix it using this:

```
// Make line stretch to whole width
line.style.width = pre.firstChild.offsetWidth;
```

**Comment by chenningg** [comment]
Fixed using this line:

```
// Make line stretch to whole width
line.style.width = pre.scrollWidth + "px";
```

**Comment by RunDevelopment** [comment]
I think this issue is fixed by #3011.

Feel free to comment on this issue and request reopening if it wasn't fixed.
